README.md

```diff
@@ -25,120 +23,168 @@ Today, the attestation model does not seek to provide information to users withi
+In addition to the computer readable version of the attestations, a natural language version is included in the attestation file in one or more languages. Providing a natural language version of the attestation strengthens the legal representation claim for the entity providing the attestation.
+
+The English version of the core privacy attestation is:
+
+Previous:
+
+> This entity that is using the Privacy Sandbox states that it will not
+use the data from the Privacy Sandbox APIs or services for the purpose
+of linking your identity across different sites or apps.
 
-The equivalent English version of the core privacy attestation would be:
+New:
 
-> This entity that is using the Privacy Sandbox states that it will not use information from Privacy Sandbox APIs or services in a way that allows that entity to identify you or your device across sites or apps. 
+> The attesting entity states that it will not use the Privacy Sandbox
+APIs or services for the purpose of learning that you are the same
+user across different sites or apps, and that it will not otherwise
+circumvent the privacy protections of the Privacy Sandbox.
 
-In the attestation file, a language marker will need to be provided for any translations of the natural language version so that a user agent or app can select the appropriate language version to display to the user if desired. Machine based translations could provide translations for other languages that are not included in the attestation file.
+In the attestation file, a language marker must be provided for any translations of the natural language version so that a user agent or app can select the appropriate language version to display to the user if desired. Machine-based translations can provide translations for other languages that are not included in the attestation file.
 
 Let's take a look at the parts that make up the proposed core privacy attestation:
+
 <table>
   <tr>
-   <td><strong>Service</strong></td>
-   <td>A 'service' reflects both client side APIs in the Privacy Sandbox and any related servers that altogether, colloquially represent an API. For example, the Protected Audience API includes both client side APIs in a browser or Android but also a variety of servers such as the key-value server, which inclusively are considered the <em>service</em>.</td>
-  </tr>
-  <tr>
-   <td><strong>NotUsed</strong></td>
-   <td>The public documented privacy goal of the Privacy Sandbox is based on protecting the user from a particular privacy abuse. By having the site provide a negative attestation via <em>NotUsed</em>,  that it is not violating that principle, the attestation can offer more flexibility for how a site can use the data from the services in new and innovative ways while still staying true to the privacy protecting principle we have laid out.</td>
+   <td><strong>Service</strong>
```

Comment:
Update to <code> formatting for the terms as they reference the code formatted attestation above.

Reply:
ah I just changed from code as the last iteration used strong. Changing back.